docs/extending/reusable_back_office_templates.md

```diff
@@ -0,0 +1,91 @@
+# Reusable Back Office templates
+
+When extending the Back Office, you can use base Twig templates for commonly used UI components such as tabs or tables.
+
+The available templates are:
+
+- `@ezdesign/ui/component/table/table.html.twig`
+- `@ezdesign/ui/component/tab/tabs.html.twig`
+
+## Table
+
+Table contains the following blocks:
+
+- `header` - headline for the table section
+- `headline` - table name
+- `actions` - action buttons, for example create, bulk delete
+- `table` - the table itself
+- `thead` - table header content
+- `tbody` - table body content
+
+The table component supports the following variables:
+
+- `table_class` - additional CSS classes attached to the `<table>` tag
+
+## Tabs
+
+Tabs contain the following blocks:
+
+- `tab_content` - tab content
+
+The tabs component supports the following variables:
+
+- `tabs`
+- `id` - tab ID
+- `label` - human-readable label for the tab
+- `active` - true if tab is active
+- `content` - HTML content of tab if `tab_content` is not overridden, 
+- `tab_content_class` - additional CSS classes attached to `.tab-content`
+- `tab_content_attributes` - additional HTML attributes added to `.tab-content`
+
+To use the components, either [`embed`](https://twig.symfony.com/doc/3.x/tags/embed.html)
+or [`include`](https://twig.symfony.com/doc/3.x/tags/include.html) them in templates.
+
+Using `embed` enables you to override blocks defined inside the included template, for example:
+
+``` html+twig
+{% embed '@ezdesign/ui/component/table/table.html.twig' %}
+    {% block headline %}
+        Headline
+    {% endblock %}
+
+    {% block actions %}
+        <a href="#" class="btn btn-icon">
+            <svg class="ibexa-icon ibexa-icon--small ibexa-icon--create">
+                <use xlink:href="{{ ez_icon_path('create') }}"></use>
+            </svg>
+        </a>
+    {% endblock %}
+    
+    {% block thead %}
+        <tr>
+            <th></th>
+            <th>Column A</th>
+            <th>Column B</th>
+            <th>Column C</th>
+        </tr>
+    {% endblock %}
+
+    {% block tbody %}
+        {% for i in 1..10 %}
+            <tr>
+                <td></td>
+                <td>A{{ i }}</td>
+                <td>B{{ i }}</td>
+                <td>C{{ i }}</td>
+            </tr>
+        {% endfor %}
+    {% endblock %}
+{% endembed %}
+```
```

Comment:
Example of table component usage should be placed under Table section :wink:

Reply:
That's what happens when you commit on Friday 😁